Packs/CofenseTriage/Integrations/CofenseTriagev3/README.md

```diff
@@ -0,0 +1,2393 @@
+The Cofense Triage v3 integration uses the Cofense Triage v2 API (previous integrations were limited to Triage v1 API) that allows users to ingest phishing reports as incident alerts and execute commands.
+
+Security teams can ingest data from Triage such as email reporters, email reports and clusters, threat indicators, and rule matching to name a few. In addition, ingest and create threat indicators, categorize reports, and obtain second stage threat indicators from malicious emails. This integration was integrated and tested with version 1.22.0 of Cofense Triage.
+
+## What's new in Cofense Triage v3?
+Use Cofense Triage v3 XSOAR integration application to take advantage of Cofense Triage v2 API for bidirectional communication between Triage and XSOAR. 
```

Comment:
this is not what i meant, we need to elaborate on the differences between the v2 and this integration
please see https://github.com/demisto/content/blob/master/Packs/VirusTotal/Integrations/VirusTotalV3/README.md#changes-from-virustotal-integration and  https://github.com/demisto/content/blob/master/Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/README.md#breaking-changes-from-previous-versions-of-this-integration for reference